docs/running.rst

```diff
@@ -123,7 +210,7 @@ The output of the regression run looks like the following:
 .. code-block:: none
 
   Command line: ./reframe.py -C tutorial/config/settings.py -c tutorial/example1.py -r
-  Reframe version: 2.13-dev0
+  Reframe version: 2.15-dev0
```

Comment:
Do not change manually the versions numbers if you don't re-reproduce the output. The date just after this output shows May 26, when the 2.15 wasn't available!